src/main/java/de/tum/in/www1/artemis/web/rest/ModelingExerciseResource.java

```diff
@@ -415,4 +410,31 @@ public ModelingExerciseResource(ModelingExerciseRepository modelingExerciseRepos
         log.info("Finished plagiarismResultRepository.savePlagiarismResultAndRemovePrevious call in {}", TimeLogUtil.formatDurationFrom(start));
         return ResponseEntity.ok(result);
     }
+
+    /**
+     * PUT /modeling-exercises/re-evaluate : Re-evaluates and updates an existing modelingExercise.
+     *
+     * @param modelingExercise     the modelingExercise to re-evaluate and update
+     * @param deleteFeedbacks  about checking if the feedbacks should be deleted when the associated grading instructions are deleted
```

Comment:
see above, use the same documentation for this variable throughout the whole associated code;  also align the beginning of the description with the start of the description above?